presto-docs/src/main/sphinx/cache.rst

```diff
@@ -0,0 +1,8 @@
+**********
+Data Cache
```

Comment:
I think we should either name this to Cache, or rename the file to datacache. I would prefer the first.

Reply:
cache might be better as later I plan to add more sections on metadata cache